lib/Bindings/Python/dialects/om.py

```diff
@@ -122,6 +134,22 @@ def __iter__(self):
       yield (wrap_mlir_object(i), self.__getitem__(i))
 
 
+class FrozenBasePath(BaseFrozenBasePath):
+
+  def __init__(self, obj: BaseFrozenBasePath) -> None:
+    super().__init__(obj)
+
+  @staticmethod
+  def get_empty(context=None) -> "FrozenBasePath":
+    return FrozenBasePath(BaseFrozenBasePath.get_empty(context))
+
+
+class FrozenPath(BaseFrozenPath):
```

Comment:
Since we don't need any pure Python enhancements over what is defined in C++, I think the body here could just be `pass`, and this can live as a placeholder for any future pure Python enhancements. Or, up top, just import `FrozenPath` without `as BaseFrozenPath` and get rid of this definition to remove the layer of indirection.

Reply:
Thanks for the suggestion! By not having to `import Path as BasePath` I was able to remove `Frozen` from the class names.